src/Symfony/Component/Mailer/Bridge/Amazon/composer.json

```diff
@@ -22,6 +22,9 @@
     "require-dev": {
         "symfony/http-client": "^4.3|^5.0"
     },
+    "suggest": {
+        "symfony/http-client": "To enable the \"ses+api\" and \"ses+https\" transports."
```

Comment:
Beware whne merging back 4.4 to 5.1, the SesApi has been deprecated in favor of `SesApiAsync`.
Starting from 5.1, we should suggest `async-aws/ses` instead of `symfony/http-client`

Reply:
I'll keep that in mind, thank you!